index.html

```diff
@@ -315,9 +315,93 @@ <h2 class="component_ex">Example of Floating action Button</h2>
                     </iframe>
                 </div>
                 <hr>
+                <!-- Image component -->
+                <div id="Image">
+                    <h1 class="component_heading">Image</h1>
+                    <p class="para">
+                        Responsive images are the set of techniques used to load the right image based on device
+                        resolution, orientation, screen size, network connection, and page layout.
+                    </p>
+                    <h2 class="component_ex">Example For Responsive Images</h2>
+                    <div>
```

Comment:
change the class name to component_example for better clarity.

Reply:
ok i will change this